src/renderer/views/Playlist/Playlist.js

```diff
@@ -243,6 +243,15 @@ export default defineComponent({
     sortBySelectValues() {
       return this.sortByValues
     },
+    totalPlaylistDuration() {
+      if (!this.playlistItems || this.playlistItems.length === 0) {
```

Comment:
This function should only be called if the length is greater than 0 so this shouldn't be necessary (247-249)